OpenRA.Mods.Common/Traits/World/MusicPlaylist.cs

```diff
@@ -162,9 +162,15 @@ void Play()
 			if (!SongExists(currentSong) || (CurrentSongIsBackground && IsBackgroundMusicMuted))
 				return;
 
+			if (Game.Settings.Sound.Repeat)
+			{
+				Game.Sound.PlayMusic(currentSong, looped: true);
```

Comment:
Is there a specific reason you use a named parameter here? Readability?

Reply:
Yes.

Reply:
Ok.